pages.id/windows/winget.md

```diff
@@ -0,0 +1,32 @@
+# winget
+
+> Manajer Paket Antarmuka Baris Perintah Windows.
+> Informasi lebih lanjut: <https://learn.microsoft.com/id-id/windows/package-manager/winget>.
+
+- Instal paket:
+
+`winget install {{nama_paket}}`
+
+- Menampilkan informasi tentang paket:
```

Comment:
following this rule #8576, it should be `Tampilkan` instead of `Menampilkan`.
cc @reinhart1010